blocks/research/visually-hidden/visually-hidden.css

```diff
@@ -0,0 +1,12 @@
+.visually-hidden {
```

Comment:
@Woolfs1423 The **BEM mix** `visually-hidden` is not an element of the research block and should be placed in its own directory within the `blocks/` directory.

Reply:
I fixed all